pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -906,7 +906,7 @@ func (gce *GCECloud) projectOwnsStaticIP(name, region string, ipAddress string)
 	return false, nil
 }
 
-func (gce *GCECloud) ensureStaticIP(name, serviceName, region, existingIP string) (ipAddress string, created bool, err error) {
+func ensureStaticIP(s CloudAddressService, name, serviceName, region, existingIP string) (ipAddress string, existing bool, err error) {
```

Comment:
I have some thoughts on what the layer should be but out of scope for this PR most likely.